cmd/root.go

```diff
@@ -1,61 +1,363 @@
 package cmd
 
 import (
-	"strings"
-
-	"os"
-
+	"context"
+	"encoding/json"
+	"errors"
 	"fmt"
+	"net/url"
+	"os"
+	"strings"
+	"time"
 
 	"github.com/Sirupsen/logrus"
+	"github.com/andygrunwald/go-jira"
 	"github.com/fsnotify/fsnotify"
+	"github.com/google/go-github/github"
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
+	"golang.org/x/oauth2"
 )
 
 var (
-	log             *logrus.Logger
+	log             *logrus.Entry
```

Comment:
It's how it was done in https://github.com/sym3tri/hookz, which is what I was told to use as a template for this project.